src/UseCaseContext.ts

```diff
@@ -16,18 +16,16 @@ export default class UseCaseContext {
     dispatcher: UseCase;
 
     /**
-     * @param   [dispatcher]    dispatcher is Dispatcher or parent UseCase.
-     * @public
+     * @param   dispatcher
+     *  The parent UseCase.
      */
     constructor(dispatcher: UseCase) {
```

Comment:
> in jsdoc, this dispatcher is marked as optional arguments.

This is wrong. `dispatcher` of the UseCaseContext's constructor is require.
([This commit](in/pull/91/commits/860dea85fe7bb9d9f36bdb679c8c5caa90a4a4a5)  is no problem)

Insteadof this, We can add `null` default value to `UseCaseExecutor` constructor.

```ts
    constructor({
        useCase,
        parent = null, // <= 
        dispatcher
    }: UseCaseExecutorArgs)
```

summarized

- `new UseCaseExecutor({ useCase, dispatcher, parent = null })` from `Context`.
- `new UseCaseExecutor({ useCase, dispatcher, parent = dispatcher})` from `UseCaseContext`.